hw2/stackCalculator/Calculator.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using StackCalculator.Tests;
+
+namespace StackCalculator
+{
+    internal class Calculator
```

Comment:
Используйте file-scoped namespaces:
```suggestion
namespace StackCalculator;

﻿﻿using System;
using StackCalculator.Tests;

internal class Calculator
```